Lib/fontbakery/specifications/general.py

```diff
@@ -751,19 +751,26 @@ def com_google_fonts_check_052(ttFont):
 
 @check(
   id = 'com.google.fonts/check/053'
+  rationale = """Some font editors store source data in their own SFNT
+  tables, and these can sometimes sneak into final release files, 
+  which should only have OpenType spec tables.""" 
 )
 def com_google_fonts_check_053(ttFont):
   """Are there unwanted tables?"""
   UNWANTED_TABLES = {
-      'FFTM', 'TTFA', 'prop', 'TSI0', 'TSI1', 'TSI2', 'TSI3', 'TSI5'}
+      'FFTM (from FontForge)', 'TTFA (from TTFAutohint)', 
+      'TSI0 (from VTT)', 'TSI1 (from VTT)', 'TSI2 (from VTT)',
+      'TSI3 (from VTT)', 'TSI5 (from VTT)'}
```

Comment:
adding the suffixes in parentheses breaks the detection of the tables. I will fetch from your branch and fix this with a correct implementation of the intention you tried to express here. I like the idea, but the implementation is incorrect.